src/ProjectTaskTemplate.php

```diff
@@ -244,21 +244,6 @@ public function displaySpecificTypeField($ID, $field = [], array $options = [])
     }
 
 
-    public static function getSpecificValueToDisplay($field, $values, array $options = [])
```

Comment:
This removed part (at least the `type` case) seems to be not present in parent

Reply:
There is no `type` specific field in search options, and deleted code cannot work as there is no `getTypes()` method in this class.